app/Http/Controllers/CampaignController.php

```diff
@@ -37,7 +39,20 @@ public function create(Request $request)
      */
     public function store(CampaignStoreRequest $request)
     {
-        $campaign = Campaign::create($request->validated());
+        $data = $request->validated();
+
+        $campaign = Campaign::create($data);
+
+        //user joins capaign
+        $user = User::find(Auth::id())->first();
```

Comment:
Theres a short hand that does this for you
```suggestion
        //user joins capaign
        $user = Auth::user()
```